bin/test.js

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env node
+import { ConfigLoaderError, readConfig } from '@web/config-loader';
+import commandLineArgs from 'command-line-args';
+import commandLineUsage from 'command-line-usage';
+import process from 'node:process';
+import { startTestRunner } from '@web/test-runner';
+import { WTRConfig } from '../src/server/wtr-config.js';
+
+const DISALLOWED_OPTIONS = ['--browsers', '--playwright', '--puppeteer', '--groups'];
+
+const optionDefinitions = [
+	// @web/test-runner options
+	{
+		name: 'files',
+		type: String,
+		multiple: true,
+		description: 'Test files to run. Path or glob.\n[Default: ./test/**/*.<type>.js]',
+		order: 8
+	},
+	{
+		name: 'group',
+		type: String,
+		required: true,
+		defaultOption: true,
+		description: 'Name of the group to run tests for\n[Default: unit]',
+		order: 1
+	},
+	{
+		name: 'manual',
```

Comment:
As discussed in standup, maybe we remove this option to discourage it, and people like me that still like it can just go `--watch`, then `m`, then `d` (which I already do now)